docs/dev/releasing.rst

```diff
@@ -59,3 +59,9 @@ Merge conda-forge version update
 The conda-forge ``regro-cf-autotick-bot`` will create a PR after
 detecting the new release on PyPI. You should be able to just merge
 the PR when CI passes.
+
+
+.. raw:: html
+
+    <script data-goatcounter="https://distdatacats.goatcounter.com/count"
```

Comment:
Reading https://distdatacats.goatcounter.com/help/domains , we need to add the host in here, or something to label this as dask-awkward

Reply:
We can try!